kolibri/plugins/coach/assets/src/views/StorageNotificationBanner.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+
+  <div class="alert" :style="{ backgroundColor: $themePalette.red.v_50, display: displayBanner }">
+    <h1 style="display: none">
+      Insufficient learner storage notification
+    </h1>
+    <div style="display:flex">
+      <div>
+        <KIcon
+          icon="error"
+          class="icon"
+        />
+      </div>
+
+      <div class="error-message">
+        {{ $tr('warningMessage') }}
+        <router-link :to="url">
+          <KButton appearance="basic-link">
+            {{ $tr('alertLink') }}
+          </KButton>
+        </router-link>
+      </div>
+
+      <div>
+        <KIconButton
+          size="small"
+          icon="incorrect"
+          :ariaLabel="coreString('closeAction')"
+          :tooltip="coreString('closeAction')"
+          @click="closeAlert"
+        />
+      </div>
+
+    </div>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import { mapState } from 'vuex';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import { useLocalStorage } from '@vueuse/core';
+  import { PageNames } from '../constants';
+
+  export default {
+    name: 'StorageNotificationBanner',
+    components: {},
+    mixins: [commonCoreStrings],
+    data: function() {
+      return {
+        displayBanner: 'auto',
+        notificationDismissal: {},
+      };
+    },
+    computed: {
+      ...mapState('classSummary', { classId: 'id' }),
+      url() {
+        return {
+          name: PageNames.PLAN_PAGE,
+          classId: this.classId,
+        };
+      },
+    },
+    beforeMount() {},
```

Comment:
This hook doesn't seem to be used. Its Probably best to remove it :)

Reply:
Good catch, I did not mean to leave that in!